assets/base/i18n/en.json

```diff
@@ -508,6 +508,15 @@
   "mobile.screen.settings": "Settings",
   "mobile.screen.your_profile": "Your Profile",
   "mobile.search.jump": "Jump to recent messages",
+  "mobile.search.modifier_after": "after a date",
+  "mobile.search.modifier_before": "before a date",
+  "mobile.search.modifier_exclude": "exclude search terms",
+  "mobile.search.modifier_from": "a specific user",
+  "mobile.search.modifier_in": "a specific channel",
+  "mobile.search.modifier_on": "a specific date",
+  "mobile.search.modifier_phrases": "messages with phrases",
+  "mobile.search.show_less": "",
+  "mobile.search.show_more": "",
```

Comment:
These don't have a text. Probably because they use the `t` function, in which case you have to manually set the text here.